e2e/vm/additional_disk_test.go

```diff
@@ -42,7 +42,9 @@ var testAdditionalDisk = func(o *option.Option, installed bool) {
 			time.Sleep(20 * time.Second)
 
 			command.New(o, virtualMachineRootCmd, "stop").WithTimeoutInSeconds(90).Run()
+			time.Sleep(10 * time.Second)
```

Comment:
Wouldn't 0.5 to 1 second be enough? Adding a delay of 10 seconds would keep us from catching any issues that come up if user tries to change VM status without long delays.

Reply:
Yeah, that's a good point. I'll bring it down to 1 second to simulate "real user" inputs. Curious to see how this run of tests turns out with 10s though